src/MapUpMap.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import Map from 'react-map-gl';
-
+import Footer from './Footer';
 // eslint-disable-next-line import/no-webpack-loader-syntax
 import maplibregl from "!maplibre-gl";
-
+import axios from "axios";
 function MapUpMap() {
+
+  let apiKey = "MHc0JQm8IOI3imNzjG0nx74AymP5P6eZ";
```

Comment:
Will need to remove this API key. It would also be a good idea to refresh the key online to keep it secret

Reply:
Sorry, I mean we can't conceptually use apiKey (not just put a placeholder).

We're treating the project as a 'real' project so everything that goes to main has to essentially be at least able to run.